Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM sameersbn/ubuntu:14.04.20161217
```

Comment:
@phenomax, the line was not redundant. There 'was' an invisible character (carriage return, I believe) leading the `FROM` instruction. The line should be added back, but with out the `^R`. You typically won't see this character on GitHub, but if you do a quick `git diff` you can tell it's there:
![image](https://cloud.githubusercontent.com/assets/2544108/21789862/1710d246-d6a6-11e6-812e-62bc8b77606f.png)